Pdf/Model/Config/Source/JapaneseFont.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: tsuchiya
+ * Date: 2016/07/01
+ * Time: 15:23
+ */
+namespace MagentoJapan\Pdf\Model\Config\Source;
+
+/**
+ * Class JapaneseFont
+ * @package MagentoJapan\Pdf\Model\Config\Source
+ */
+class JapaneseFont implements \Magento\Framework\Option\ArrayInterface
+{
+    /**
+     * font file path
+     */
+    const FONT_DIR = 'lib/fonts/';
```

Comment:
There is already a similar constant used in the `Helper`. It would make sense to make it configurable from a single point.